sdk/core/Azure.Core/src/SerializableOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure
+{
+    /// <summary>
+    /// TODO
+    /// </summary>
+    public class SerializableOptions
+    {
+        /// <summary>
+        /// TODO
+        /// </summary>
+        public bool IncludeReadOnlyProperties { get; set; } = true;
+
+        /// <summary>
+        /// TODO
+        /// </summary>
+        public bool HandleAdditionalProperties { get; set; } = true;
```

Comment:
Nit: .NET guidelines are that properties should typically default to `false` (or `default`, in general). Perhaps these should be renamed `ExcludeReadOnlyProperties` and `ExcludeAdditionalProperties`. This is the reason we have `Exclude*` in Azure.Identity options.

Reply:
double negatives are always confusing, prefer if we keep these as positive statements instead of at all possible.

Reply:
Do we have a crisp definition of "AdditionalProperties"?  We might want to tune this naming for clarity.  Is it a "Missing Member", per [this definition](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/missing-members)?

Reply:
+1 "IgnoreReadOnlyProperties", to be consistent with [BCL naming](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions.ignorereadonlyproperties)

Reply:
@JoshLove-msft I like that suggestion @nisha-bhatia  lets add a task to rename both to Ignore and default to false.

@annelo-msft I think a missing member is the other way around, the class expected it but the payload didn't include.  AddtionalProperties are extra things that the class doesn't know about.